images/Dockerfile-clientserver

```diff
@@ -1,6 +1,7 @@
 FROM quay.io/redhat-user-workloads/rhtas-tenant/cli-1-0-gamma/cosign-cli-2-2@sha256:83e03294d6dfe8443762a735b2abb1171043cbfb93694a459e4432e520acf9a2 AS cosign-image
 FROM quay.io/redhat-user-workloads/rhtas-tenant/cli-1-0-gamma/gitsign-cli-0-8@sha256:605ee6d943acadd2ad45322f9ac49651524ffaae1dab64bfd251cddae49b86aa AS gitsign-image
 FROM quay.io/redhat-user-workloads/rhtas-tenant/cli-1-0-gamma/rekor-cli-1-3@sha256:b94954f6f82d9e5743c368a974a9d96c640e1ca7ba985fd3a86df5a989b8dd12 AS rekor-image
+FROM quay.io/redhat-user-workloads/rhtap-contract-tenant/ec-v01-alpha/cli-v01-alpha@sha256:961de8f1a62977068fb2f654640d14e5eb42427366913f3b6aa7690f9b45c1bd AS ec-image
```

Comment:
Why are we pulling from this location? Because it is not released?

Reply:
Good question, we could probably switch to registry.redhat.io once the release does go out. But right now in this PR I'm trying to be consistent with the other cli images.

Reply:
In fact, I believe all of the other images should be registry.redhat.io and QE should be testing with an `ImageDigestMirrorSet` to proxy to quay.io for pre-release images.

Reply:
I think that change should happen in a separate PR, perhaps after this one, though it could be done before I guess.

Note that the ec image hasn't landed in registry.redhat.io yet, (though it might get pushed there any minute now).